.travis.yml

```diff
@@ -34,15 +34,7 @@ global:
 addons:
   apt:
     packages:
-      # - dosbox
```

Comment:
Okay, done.

There is a sudo for openwatcom, too, unfortunately, which won't go away unless an apt package were created for openwatcom, or we need to find a way to install the package for the local user. I hear this is possible but somewhat involved.